about.html

```diff
@@ -0,0 +1,153 @@
+        </div>
+        <div class="about-section__containers">
+          <p class="about-section__description">The logo of Best games! 2015 was decided through the logo competition from 8 June to 7 July</p>
+        </div>
+        <div class="about-section__containers">
+          <img class="about-section__logo" src="./images/icons/mainIcon.png">
+        </div>
+      </div>
+      <div class="about-section__titles">
+        <div class="about-section__containers">
+          <h2 class="about-section__main-title">See the best rated games ever</h2>
+          <div class="orange-line"></div>
+        </div>
+        <div class="about-section__containers">
+          <p class="about-section__description">Take a look at the two best seller games not only in Best games! but worldwide</p>
+        </div>
+      </div>
+      <div class="about-section__images">
+        <div class="about-section__image-container container-solo">
+          <div class="about-section__orange">
+            <p class="about-section__title-one">Until Dawn</p>
+            <p class="about-section__title-two">Best rated game</p>
+          </div>
+          <img class="about-section__image" src="./images/skullBackground.jpg">
+        </div>
+        <div class="about-section__image-container">
+          <div class="about-section__orange">
+            <p class="about-section__title-one">Skyrim</p>
+            <p class="about-section__title-two">Second best rated game</p>
+          </div>
+          <img class="about-section__image" src="./images/skyrimBAckground.jpg">
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <!-- Footer section -->
+
+  <footer class="footer-section">
+    <div class="footer-section__container">
+      <div class="footer-section__first-part">
+        <div class="footer-section__title-container">
+          <p class="footer-section__title">Partner</p>
+          <div class="orange-line"></div>
+        </div>
+        <div class="footer-section__list-container">
+          <ul class="footer-section__first-logos">
+            <li><img class="footer-section__image" src="./images/mozillaL.png"></li>
+            <li><img class="footer-section__image" src="./images/pngfind.com-flat-google-logo-png-4206174.png"></li>
+            <li><img class="footer-section__image naver" src="./images/naverLogo.png"></li>
+          </ul>
+          <ul class="footer-section__last-logos">
+            <li><img class="footer-section__image kakao" src="./images/kakao.png"></li>
+            <li><img class="footer-section__image" src="./images/airBnbLogo.png"></li>
+          </ul>
+        </div>
+      </div>
+      <div class="footer-section__second-part">
+        <div class="footer-section__logo-container"><img class="footer-section__logo" src="./images/icons/mainIcon.png"></div>
+        <div class="footer-section_text-container"><p class="footer-section__small-text">Buy games!<br>Some Rights Reserved</p></div>
+      </div>
```

Comment:
- [ ] Kindly adjust the text to be aligned to the right just as done on the original design.
- This makes your work look perfect and solidifies your attention to details 
- In this `about` section, the background is black, not white. Kindly fix this too 👍 

## Your work:
![image](https://github.com/EvansSnave/capstone-project/assets/50990925/719d3eef-005b-4d0f-a545-5065ac83462a)

 ## Design given:
![image](https://github.com/EvansSnave/capstone-project/assets/50990925/efe305a3-fb86-48f9-abe2-a05e6bd07ea1)

## Design in about page:
![image](https://github.com/EvansSnave/capstone-project/assets/50990925/a7f97b02-cde1-4b96-abbf-cfb145a892bb)